src/elm/Build.elm

```diff
@@ -523,15 +570,15 @@ toTwoDigits int =
 stepClasses : Step -> Steps -> Html.Attribute msg
 stepClasses step steps =
     let
```

Comment:
this fix is needed because I added sorting in steps (server returns them reversed)